### DISCUSSION TITLE
Suggestion to update README

### DISCUSSION THREAD

**Discussion by minminlittleshrimp** [COVESA/dlt-daemon#465] [Ideas]
Hello @michael-methner 

Currently lgtm.com is no more available and the team becomes part of Github, they are focusing on Github Actions.
For DLT Daemon badges update and code vulnerability, code coverage checking, I have some suggestions here:
- Vulnerability is checked by [CodeQL](https://github.com/COVESA/dlt-daemon/actions/workflows/codeql-analysis.yml) already integrated into Github Actions.
- Code coverage measurement could be done by integrating [codecov](https://about.codecov.io/) into Github Actions.

What do you think?
Regards

**Comment by michael-methner** [comment]
Hello @minminlittleshrimp ,
thanks for the suggestion. Have removed LGTM and added CodeQL here:
https://github.com/COVESA/dlt-daemon/pull/467

codecov wants to have access to my private repositories. So I think it is not compatible with account ;)

**Comment by michael-methner** [comment]
Hello @minminlittleshrimp ,
if you find a alternative for coverage measurement, I would be happy to include the badge.

**Comment by minminlittleshrimp** [comment]
Hello @michael-methner 
I would like to close this discussion due to successful implementation of gcov/lcov for code coverage of dlt-daemon.
Thank you
